src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditOrganisationDetails.cshtml

```diff
@@ -50,7 +50,7 @@
         </div>
         <div class="govuk-grid-row govuk-!-padding-bottom-2">
             <div class="flex-container">
-                <span class="govuk-body govuk-label--s label-width" for="Organisation_CompaniesRegistrationNumber" id="Organisation_CompaniesRegistrationNumber-label">Company registration number</span>
+                <span class="govuk-body govuk-label--s label-width" for="Organisation_CompaniesRegistrationNumber" id="Organisation_CompaniesRegistrationNumber-label">company registration number</span>
```

Comment:
Dont think this should change as its heading.

Reply:
restored